src/label_studio_sdk/converter/imports/coco.py

```diff
@@ -48,10 +48,9 @@ def create_bbox(annotation, categories, from_name, image_height, image_width, to
 
 
 def create_segmentation(
-    annotation, categories, from_name, image_height, image_width, to_name
+    category_id, segmentation, categories, from_name, image_height, image_width, to_name
```

Comment:
due to the change from this PR, there's no need to pass the whole annotation.
so updated to pass `annotation["category_id"]` and `segmentation`